code/game/objects/items/weapons/trays.dm

```diff
@@ -1,6 +1,10 @@
 /*
- * Trays - Agouri
+ * Trays - Nanako

```

Comment:
This entire file is basically thrown out and rewritten, not much point in looking at a comparison, read the whole thing

only part i didn't really touch was its combat interactions